examples/layers/preprocessing/random_shear_bbox_demo.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""random_shear_bboxdemo.py shows how to use the RandomShear preprocessing layer
+   for object detection.
+Operates on the voc dataset.  In this script the images and bboxes
+are loaded, then are passed through the preprocessing layers.
+Finally, they are shown using matplotlib.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv.layers import preprocessing
+
+IMG_SIZE = (256, 256)
+BATCH_SIZE = 9
+
+
+def resize(inputs):
+    """mapping function to create batched image and bbox coordinates"""
+    inputs["image"] = tf.image.resize(inputs["image"], IMG_SIZE)[0]
+    height, width, _ = inputs["image"].shape
+    y1, x1, y2, x2 = tf.split(inputs["objects"]["bbox"][0], 4, axis=1)
+    inputs["objects"]["bbox"] = tf.squeeze(
+        tf.stack(
+            [
+                x1 * (IMG_SIZE[0] / width),
+                y1 * (IMG_SIZE[1] / height),
+                x2 * (IMG_SIZE[0] / width),
+                y2 * (IMG_SIZE[1] / height),
+            ],
+            axis=1,
+        ),
+        axis=-1,
+    )
+    return inputs
+
+
+def _rel_xyxy_to_rel_yxyx(bounding_boxes, height, width):
+    x1, y1, x2, y2 = tf.split(bounding_boxes, 4, axis=2)
+    new_bboxes = tf.squeeze(
+        tf.stack(
+            [
+                y1,
+                x1,
+                y2,
+                x2,
+            ],
+            axis=2,
+        ),
+        axis=-1,
+    )
+    return new_bboxes
+
+
+def main():
+    dataset = tfds.load(
+        "voc/2007", split=tfds.Split.TRAIN, batch_size=1, shuffle_files=True
```

Comment:
we can add this to the demo utils:

`examples/layers/preprocessing/demo_utils.py`

Reply:
Sure, You mean creating a padded batch?

Reply:
are the paddings 0s or -1s?  For consistency with COCO metrics I'd like to use -1s

Reply:
Sure will pass padding_values as -1 rather than default

Reply:
Done